packages/flutter/lib/src/material/text_field.dart

```diff
@@ -849,6 +851,18 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     super.deactivate();
   }
 
+  void _handlePointerEnter(PointerEnterEvent event) => _handleHover(true);
+  void _handlePointerExit(PointerExitEvent event) => _handleHover(false);
+
+  void _handleHover(bool hovering) {
+    if (hovering != _isHovering) {
+      setState(() {
+        print('Hovering: $hovering');
```

Comment:
Did you intend for this `print` to stay in here?

Reply:
Nope.  Removed.